walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -121,6 +123,63 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void notValidOptionalAttendee(){
```

Comment:
Optional but recommended: Format all test method names according to [styleguide's](https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names) typical pattern. I've been burned on readability reviews a few times from this so it's good to keep in mind, despite it being a loose style restriction.

Reply:
What is the 'methodUnderTest' because right now I'm only testing one method and it might be redundant restating it each time.

Reply:
The appropriate method here would be 'query'. Its okay to leave it out for now, but you'll want to put it in once you have more methods being tested, so might not hurt to have it in early if you plan to add more methods to a class.